Test/comp/DowncastDt.dfy

```diff
@@ -8,22 +8,32 @@
 // Java avoids this problem by not even allowing traits in datatypes.
 
 datatype Co<+T> = Co(T)
+datatype Con<-T> = Con(seq<T>)
```

Comment:
Just a quick comment, I'm sure it works, but why did you use a `seq<T>` instead of just `T`?